cmd/cli/uninstall_mesh.go

```diff
@@ -90,6 +107,7 @@ func (d *uninstallMeshCmd) run() error {
 			out:       d.out,
 			config:    d.config,
 			clientSet: d.clientSet,
+			localPort: d.localPort,
```

Comment:
I am including this fix to prevent the SMI SUPPORTED field from showing `Unknown` when the `osm uninstall mesh` command is run.

Before including this fix (this line). Note how the prompt shows `SMI SUPPORTED - Unknown`

```console
~/Dev/osm(uninstall-cluster-wide-resources ✔) ./bin/osm uninstall mesh

List of meshes present in the cluster:

MESH NAME   MESH NAMESPACE   VERSION       ADDED NAMESPACES
osm         osm-system       latest-main   bookbuyer,bookstore,bookthief,bookwarehouse

MESH NAME   MESH NAMESPACE   SMI SUPPORTED
osm         osm-system       Unknown

To list the OSM controller pods for a mesh, please run the following command passing in the mesh's namespace
	kubectl get pods -n <osm-mesh-namespace> -l app=osm-controller

Uninstall OSM [mesh name: osm] in namespace [osm-system] ? [y/n]: y
OSM [mesh name: osm] in namespace [osm-system] uninstalled
```


After including this fix/this line. Notice how the SMI SUPPORTED versions are now being displayed properly.

```console
~/Dev/osm(uninstall-cluster-wide-resources ✔) ./bin/osm uninstall mesh

List of meshes present in the cluster:

MESH NAME   MESH NAMESPACE   VERSION       ADDED NAMESPACES
osm         osm-system       latest-main   bookbuyer,bookstore,bookthief,bookwarehouse

MESH NAME   MESH NAMESPACE   SMI SUPPORTED
osm         osm-system       HTTPRouteGroup:v1alpha4,TCPRoute:v1alpha4,TrafficSplit:v1alpha2,TrafficTarget:v1alpha3

To list the OSM controller pods for a mesh, please run the following command passing in the mesh's namespace
	kubectl get pods -n <osm-mesh-namespace> -l app=osm-controller

Uninstall OSM [mesh name: osm] in namespace [osm-system] ? [y/n]: y
OSM [mesh name: osm] in namespace [osm-system] uninstalled
```

Reply:
wasn't this fixed previously ?

Reply:
I think my previous fix missed this fix here...